snuba/migrations/snuba_migrations/metrics/0020_polymorphic_buckets_table.py

```diff
@@ -0,0 +1,74 @@
+from typing import Sequence
+
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.utils.schemas import Array, Column, DateTime, Float, Nested, String, UInt
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    """
+    Create a polymorphic bucket table for metrics_sets,
+    metrics_distributions, and metrics_counters
+    """
+
+    blocking = False
+    local_table_name = "metrics_buckets_consolidated"
```

Comment:
So first you need to add a `_local` suffix for the local table. That's the convention we use everywehere.
Then I would call it `metrics_raw_local`. The aggregated tables do not have a suffix that tell that they are aggregated, while the raw/bucketed one do. So this is a bucketed one I would add a suffix to say that. We cannot call this `metrics_buckets_local` because it is already taken, so maybe `raw` ?